dgraph/Makefile

```diff
@@ -17,15 +17,15 @@
 USER_ID         = $(shell id -u)
 BIN             = dgraph
 BUILD          ?= $(shell git rev-parse --short HEAD)
-BUILD_CODENAME  = rocket-2
+BUILD_CODENAME  = unnamed
```

Comment:
I think we should drop the `BUILD_CODENAME` variable completely. We won't be using it going forward.